framework/src/modules/interoperability/mainchain/endpoint.ts

```diff
@@ -76,6 +82,12 @@ export class MainchainInteroperabilityEndpoint extends BaseInteroperabilityEndpo
 			);
 		}
 
+		if (name.length < MIN_CHAIN_NAME_LENGTH || name.length > MAX_CHAIN_NAME_LENGTH) {
```

Comment:
these checks can go inside schema validation, and also its better to have it in general to validate the params here